controllers/managedocs_controller_test.go

```diff
@@ -640,7 +651,7 @@ var _ = Describe("ManagedOCS controller", func() {
 			})
 			When("there is a valid size in the add-on parameter secret", func() {
```

Comment:
Why does this case have 3 different its that validate the same "should create reconciled resources" with a different skip?

To me is seems like what we want is a single "it" with an internal switch statement for the difference details